api/listings.py

```diff
@@ -91,6 +91,25 @@ def by_area(self, bbox, limit=None, **params):
 
     @cherrypy.expose
     @cherrypy.tools.json_out()
+    def by_ids(self, ids, limit=None, **params):
+        i = ids.split(',')
+        limit = self.num_param(limit, 20, 100)
```

Comment:
The limit parameter does not make sense here as we know exactly know how many results will be returned. Cut the 'i' array at 100.